### ISSUE TITLE
Consider "tart" for artwork

### ISSUE THREAD

**Issue by jtdaugherty** [ChrisPenner/void-space#1]
Hey, I thought I'd just let you know in case you haven't seen it: this tool can be used to draw and paint `brick`-compatible artwork that you can then incorporate into your `brick` application.

https://github.com/jtdaugherty/tart

It can also import plain text files (e.g. `figlet` output) for painting. :)

(Feel free to close; I just thought this would be an easy way to let you know. Very cool project!)

**Comment by jtdaugherty** [comment]
(If you want to incorporate it into your application, use `tart`-the-binary to make the artwork, then use `tart`-the-library to embed it in your `brick` UI.)

**Comment by jtdaugherty** [comment]
(In particular, `Tart.Canvas.canvasLayersToImage`.)

**Comment by ChrisPenner** [comment]
@jtdaugherty  Yeah this is super cool! I was playing around with it earlier 😄 I should try that out!
